containers/tools/install-ansible-container.yml

```diff
@@ -0,0 +1,40 @@
+---
+- hosts: all
+  gather_facts: no
+  vars:
+    project_dir: "/root"
+  tasks:
+    - name: 'Install pip'
+      yum:
+        name: python-pip
+        state: present
+
+    - name: 'Update setuptools'
+      pip:
+        name: setuptools
+        state: latest
+
+    - name: 'Install git'
+      yum:
+        name: git
+        state: present
+
+    - name: 'Create .tmp directory'
+      file:
+        path: "{{ working_dir }}/.tmp"
+        state: directory
+
+    - name: 'Clone ansible-container'
+      git:
+        repo: https://github.com/ansible/ansible-container.git
+        dest: "{{ working_dir }}/.tmp/ansible-container"
+        update: yes
+        force: yes
+
+    - name: 'Install ansible-container'
+      command: "pip install -e .[docker,openshift] --upgrade"
+      args:
+        chdir: "{{ working_dir }}/.tmp/ansible-container"
+
+    - name: 'Install updated openshift'
+      command: 'pip install openshift ansible --upgrade'
```

Comment:
Could this be done using the pip provider with state: latest?

Reply:
This can actually be removed now after I updated this dependency in a-c.